Sunburst37/ALARM_AND_ABORT.agc

```diff
@@ -94,19 +67,22 @@ JETABORT	TC	ALARM
 
 # CALLING SEQUENCE,
 
-#		TC	ALARM
-#		OCT	AAANN		ALARM NO. NN IN GENERAL AREA AAA.
+		TC	ALARM
+		OCT	AAANN		# ALARM NO. NN IN GENERAL AREA AAA.
```

Comment:
The original code was right here, these entire lines are comments. You can tell because there's an "A" in front of the punchcard number on both lines -- "A", "R", and "P" are all comment-type cards.